Libraries/LibWeb/HTML/EventNames.h

```diff
@@ -28,6 +28,14 @@ namespace Web::HTML::EventNames {
     __ENUMERATE_HTML_EVENT(blocked)                  \
     __ENUMERATE_HTML_EVENT(blur)                     \
     __ENUMERATE_HTML_EVENT(cancel)                   \
+    __ENUMERATE_HTML_EVENT(onbeforeinput)            \
+    __ENUMERATE_HTML_EVENT(onbeforetoggle)           \
+    __ENUMERATE_HTML_EVENT(oncopy)                   \
+    __ENUMERATE_HTML_EVENT(oncut)                    \
+    __ENUMERATE_HTML_EVENT(onpaste)                  \
+    __ENUMERATE_HTML_EVENT(onpageswap)               \
+    __ENUMERATE_HTML_EVENT(onpagereveal)             \
+    __ENUMERATE_HTML_EVENT(onscrollend)              \
```

Comment:
Please add these in alphabetical order; also applies to the other files you've modified.